sklearn/tests/test_kernel_approximation.py

```diff
@@ -112,11 +113,6 @@ def test_skewed_chi2_sampler():
     assert np.isfinite(kernel_approx).all(), \
         'NaNs found in the approximate Gram matrix'
 
-    # test error is raised on when inputs contains values smaller than -c
```

Comment:
We should be able to remove this since now we test for positivity and it's already tested by `check_fit_non_negative` ? Although now that I think about it `check_fit_non_negative`, would only test the `fit` part, whereas the below would test the `transform` part, so I'll let it here but adapt it to check just positivity (and not > -c). 
But then it reminds me: should we have a `check_transform_non_negative` as well in common tests ?

Reply:
Done

Reply:
It's not supposed to raise an error on negative values, only on values smaller than ``-c``.